manifest.yml

```diff
@@ -13,6 +13,7 @@ applications:
     services:
       - ((app_name))-db        # cf create-service aws-rds micro-psql ((app_name))-db
       - ((app_name))-redis     # cf create-service aws-elasticache-redis redis-dev ((app_name))-redis
+      - ((app_name))-secrets   # cf cups ${app_name}-secrets -p "BEAKER__SESSION__SECRET"
```

Comment:
FWIW, I would leave the comments out since they might get out of sync with the README.